index.html

```diff
@@ -1,29 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
     <title>Document</title>
-</head>
-<body>
-    <h1 class="heading">All Awesome Books</h1>
-    <div class="showBooks" id="showBooks"></div>
-    <hr class="hr-line">
-    <div class="container">
+  </head>
+  <body>
+    <header>
+      <nav>
+        <p>Awesome Books</p>
+        <ul class="link">
+          <li><a href="#" class="list-nav">List</a></li>
+          <li>|</li>
+          <li><a href="#" class="books-nav">Add new</a></li>
+          <li>|</li>
+          <li><a href="#" class="contact-nav">Contact</a></li>
+        </ul>
```

Comment:
- [ ] Last but not least, let's ensure that our navigation links have the `isactive` class to ensure that a link is styled differently from the rest when it is active. Active means it points us to what section of the App we are currently in.

  Looking at the wireframes we'll realize that the active links are denoted by changing their text color to blue. We can realize that by looking at the screenshots below and making the necessary adjustments:
  
  | Active links color from the wireframe | Active links don't change in the implementation|
  | --- | --- |
  | ![image](https://user-images.githubusercontent.com/45002654/153283484-6831ec54-39f3-4b02-8fe5-9541f0d5ec26.png) ![image](https://user-images.githubusercontent.com/45002654/153283502-eb323ead-9ae0-4d94-8293-44e82125c0e1.png) ![image](https://user-images.githubusercontent.com/45002654/153283522-81c56d7a-8804-465e-9edd-8625f1359c43.png) | ![image](https://user-images.githubusercontent.com/45002654/153283695-67c4e9ed-807b-42f9-83e7-76d5e4ebb685.png) |

Reply:
Thanks for your review @Lameck1 . We have worked on that and solved it.